.github/workflows/build-run-kernel.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   build-and-run-kernels:
     runs-on: ubuntu-latest
     container: 
-      image: ghcr.io/kuleuven-micas/snax:v0.1.4
+      image: ghcr.io/kuleuven-micas/snax:v0.1.6
```

Comment:
I'd prefer updating all CI routines in one go to use the newest container.
Can you also update the other CI jobs please? Thanks!